src/context.h

```diff
@@ -388,6 +388,15 @@ class TOGGL_INTERNAL_EXPORT Context : public TimelineDatasource {
         const Poco::UInt64 project_id,
         const std::string &project_guid);
 
+    error updateTimeEntry(
```

Comment:
this is a public method so it should be capitalized

Reply:
Thanks for pointing out 👍